pyuvdata/uvbeam/uvbeam.py

```diff
@@ -3986,7 +3988,9 @@ def read_cst_beam(
         fix_auto_power : bool
             For power beams, if auto polarization beams with imaginary values are found,
             fix those values so that they are real-only in data_array.
-
+        swap_thetaphi_conventions : bool
+            Whether to swap the standard assumed convention on theta/phi. By default,
+            phi is assumed to be synonymous with azimuth, and theta with zenith angle.
```

Comment:
This needs to be changed to match the actual new parameter. Also, there should be a blank line before the end of the docstring.

Reply:
Done, and added the parameter to all the other `read` docs